README.md

```diff
@@ -1,24 +1,169 @@
+- Created table to store data.
+- Created API endpoints to interact with the database.
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Data Base</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Rails</li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Use of endpoints]**
+- **[Perform CURD operation on database]**
+- **[API Usage]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+## React Project Link <a name="react-link"></a>
+https://github.com/jonas-45/motorcyle-booking
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+Will be available shortly...
+
+
+### Video Demo Link <a name="live-demo-link"></a>
+Will be available shortly...
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- You'll need any code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git@github.com:jonas-45/motorcyle-booking.git
+  cd motorcyle-booking
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+## 👥 Authors <a name="authors"></a>
+
+🕵🏽‍♀️ **Ahmad Saeed**
+
+- Github: [GitHub](https://github.com/ehmaddd/)
+- Twitter: [Twitter](https://twitter.com/ehmaddd_pk)
+- Linkedin: [LinkedIn](https://www.linkedin.com/in/ehmaddd/)
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Improve UI design]**
+- [ ] **[Making more dynamic]**
+- [ ] **[Deploy online]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jonas-45/motorcyle-booking/issues).
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give it a star!
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse community for helping us to improve our skills by providing such dynamic applications to build. After going through it, we are more than confident that we can write world-class code and implement almost any type of web app.
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- FAQ (optional) -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/jonas-45/motorcyle-booking/blob/dev/LICENSE) licensed.
+
```

Comment:
Please, update the license link. currently it is not accessible.
![image](https://github.com/jonas-45/motorcyle-booking/assets/115032263/1e81bb41-69ad-418d-9fce-ad902ecbd88c)

Reply:
Done as instructed